tests/Casts/JsonTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests\Casts;
+
+use MongoDB\Laravel\Tests\Models\Casting;
+use MongoDB\Laravel\Tests\TestCase;
+
+use function json_encode;
+
+class JsonTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        Casting::truncate();
+    }
+
+    public function testJson(): void
+    {
+        $model = Casting::query()->create(['jsonValue' => ['g' => 'G-Eazy']]);
```

Comment:
I don't think JSON cast is used with MongoDB. It's amazing to store stringified JSON in a MongoDB field.

But since Eloquent offers it, we also want it to work with this package.